### ISSUE TITLE
Request: offer to use CTRL+C for copying the content of the captured image

### ISSUE THREAD

**Issue by AndroidDeveloperLB** [ShareX/ShareX#3495]
Here:

![image](https://user-images.githubusercontent.com/5357526/42133338-0109f47c-7d31-11e8-80be-87c0a0bf5411.png)

We have copy-to-clipboard, but its shortcut is non-standard (CTRL+SHIFT+C), instead of CTRL+C.

Please allow to use CTRL+C instead.

**Comment by AndroidDeveloperLB** [comment]
Why would I want to copy a shape? It's a screenshot app...
Anyway, what happens if no shape is selected for copying? Why wouldn't it copy entire screenshot?

**Comment by AndroidDeveloperLB** [comment]
On which app do you see that CTRL+C copies a shape instead of the content or the selected content?

**Comment by diegoteliz** [comment]
I'm agree with @AndroidDeveloperLB 

Usually after you copy something, then you'll paste it somewhere else, and the shortcut on any app would be `CTRL + V`. So having `CTRL + C` and not `CTRL + SHIFT + C` would have much more sense.

I've been using the app for more than a month (everyday) and I just don't get used to it...

(btw, I get the point about using `CTRL+C` to copy shapes, but current behavior if you don't have any shape selected is to copy the coordinates. Maybe considering that as alternative?)

**Comment by noselasd** [comment]
Any update on this ? 100% of the use I have with sharex is to copy the image elsewhere (mail/documentation), and having copy be a non standard shortcut is quite a big negative with this app compared to other screenshot apps.

**Comment by AndroidDeveloperLB** [comment]
Please at least bring us the option to choose. It's frutstrating me every time I try it.
Because of this, I try to avoid all hotkeys within the app, and use only the mouse, as I don't trust it to do what's most intuitive...

**Comment by Pikachu920** [comment]
if anyone else is interested in this functionality, I wrote an [autohotkey](https://www.autohotkey.com/) script to accomplish it:
```
#NoEnv  ; Recommended for performance and compatibility with future AutoHotkey releases.
; #Warn  ; Enable warnings to assist with detecting common errors.
SendMode Input  ; Recommended for new scripts due to its superior speed and reliability.
SetWorkingDir %A_ScriptDir%  ; Ensures a consistent starting directory.

#Persistent
SetTitleMatchMode, 1
Return

#If WinActive("ahk_exe ShareX.exe") and WinActive("ShareX - Image editor -")
^C::
Send ^+c
Return
```
It just sends ctrl+shift+c in place of ctrl+c when the sharex image editor is the active window

**Comment by GoldenMaximo** [comment]
@Jaex should re-consider adding this as an option in the future.
At the very least there should be an option to customize this binding.
It would only make sense for an screenshot app as I highly doubt there are more users who care about copying coordinates than screenshots.
Lots of people mentioned this issue before and I'm sure I won't be the last one.

**Comment by Zennos** [comment]
I think this should be re-considered.
I use this all the time in others screenshot app.

> Select region
> Ctrl + C
> Region selection closes automatically
> Done!

Then I go to where I want to paste then press Ctrl + V.
I never have used a copy of a shape for anything, so would be cool if had an optional option to copy the image instead of a shape.

( Also automatically close region selection after pressing Ctrl + C )

**Comment by mystcreater** [comment]
I think there should be an option to configure hotkeys from the editor in which case "ctrl+c" or "ctrl+shift+c" would be a simple option configured by the user.

Currently, it's a brain switch for me to think using "ctrl+shift+c" instead of "ctrl+c".

I agree we rarely copy shapes on the screen. At least, it's not a reflex. I usually draw another shape in this case.

**Comment by rcastillo-developer** [comment]
Please switch these two commands. We want to copy images using Ctrl+C and copy shapes using Ctrl+Shift+C.

Any one who have made this possible in an easier way would be much appreciated if its shared. Thank you.

**Comment by c1p0** [comment]
I'm fairly new to ShareX, but I've managed to configure my workflow almost entirely except for this. CTRL + C to copy the image directly to clipboard just makes sense and there should be an option to have it PLEASE and THANK YOU!!!!

**Comment by c1p0** [comment]
> if anyone else is interested in this functionality, I wrote an [autohotkey](https://www.autohotkey.com/) script to accomplish it:
> 
> ```
> #NoEnv  ; Recommended for performance and compatibility with future AutoHotkey releases.
> ; #Warn  ; Enable warnings to assist with detecting common errors.
> SendMode Input  ; Recommended for new scripts due to its superior speed and reliability.
> SetWorkingDir %A_ScriptDir%  ; Ensures a consistent starting directory.
> 
> #Persistent
> SetTitleMatchMode, 1
> Return
> 
> #If WinActive("ahk_exe ShareX.exe") and WinActive("ShareX - Image editor -")
> ^C::
> Send ^+c
> Return
> ```
> 
> It just sends ctrl+shift+c in place of ctrl+c when the sharex image editor is the active window

I managed to accomplish this with the Keyboard Manager from PowerToys.

**Comment by Tim567** [comment]
> I managed to accomplish this with the Keyboard Manager from PowerToys.

![image](https://user-images.githubusercontent.com/24894554/222559834-a0a20a87-c594-4fa2-8138-bdbb13747098.png)

Just add this in PowerToys and you are done.

**Comment by ParadiseFallen** [comment]
there must be configurable hotkeys for editor. So anyone will configure as want.

**Comment by snowman** [comment]
what I missed is the copy(`Ctrl-C`) of selection/region without `crop`, like `fscapture` does

**Comment by pookdeveloper** [comment]
It would be nice to be able to configure the access keys for the editing part, thanks

**Comment by diegoteliz** [comment]
Great to see this one finally implemented on [v16.0.1](https://github.com/ShareX/ShareX/releases/tag/v16.0.1) 🎉

Thank you guys 👏

**Comment by AndroidDeveloperLB** [comment]
@diegoteliz For a moment I thought I need to change some settings.
It is this way now by default, right?

**Comment by diegoteliz** [comment]
> @diegoteliz For a moment I thought I need to change some settings. It is this way now by default, right?

Yes, I just updated from v15.0.0 to v16.0.1 and now it copies with CTRL+C by default.

**Comment by AndroidDeveloperLB** [comment]
@diegoteliz Nice . Thank you all !

**Comment by m-alice** [comment]
> Yes, I just updated from v15.0.0 to v16.0.1 and now it copies with CTRL+C by default.
how do i re-train my reflexes to use it and not ctlr+shift+c? ;_(

**Comment by asyncmeow** [comment]
Could an option be added to revert to the old behavior? I worked around it with a PowerToys keyboard remap to go back to ctrl+shift+c, but having this as a native ShareX option would be best IMO.

Or in other words, [could an option be added to re-enable spacebar heating?](https://xkcd.com/1172/)

**Comment by magnumpci** [comment]
> Could an option be added to revert to the old behavior? I worked around it with a PowerToys keyboard remap to go back to ctrl+shift+c, but having this as a native ShareX option would be best IMO.
> 
> Or in other words, [could an option be added to re-enable spacebar heating?](https://xkcd.com/1172/)

Indeed.  Took a google search to find out why what had been working, Ctrl + C to copy a laid down ShareX control (speech bubble for example), was now just closing the ShareX capture annotate window...  

It was rather upsetting that this was the solution, that Ctrl + C now just closes the damn ShareX annotate window - hell, I thought it crashed. Nowt I have to figure how to get it working the way it was initially intended and appreciated.

**Comment by Rozenbahs** [comment]
> I think this should be re-considered. I use this all the time in others screenshot app.
> 
>     1. Select region
> 
>     2. Ctrl + C
> 
>     3. Region selection closes automatically
> 
>     4. Done!
> 
> 
> Then I go to where I want to paste then press Ctrl + V. I never have used a copy of a shape for anything, so would be cool if had an optional option to copy the image instead of a shape.
> 
> ( Also automatically close region selection after pressing Ctrl + C )

I have had ShareX on my system for over a year as a competitive app for Lightshot app. ShareX has lots of nice features BUT I cannot get accustomised to the UX where selecting the area 1) makes the printscreen task and 2) does not allow CTRL+C as the way of finishing the task instead and 3) no way to fine-tune the selection. The proposed scenario of 1) selecting the region + 1A) do additional edits 2) CTRL+C to copy and finish the task seems to be way more intuitive.

Maybe it is there hidden deep in the settings of ShareX?

If not, could this be made as an alternative for the UX of ShareX?

**Comment by AndroidDeveloperLB** [comment]
Now with the new version, this came back with the other shortcut.
Please bring CTRL+C back in the image editor, or make it an option in the settings...
